src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -224,12 +241,52 @@ public int getVtgatePort() {
         return getConfig().getInteger(VTGATE_PORT);
     }
 
+    private void checkVtgateAuthFile() {
+        if (_username != null || _password != null) {
+            return;
+        }
+        String authFile = getVtgateAuthFile();
+        if (authFile != null && !authFile.isEmpty()) {
+            JSONParser parser = new JSONParser();
```

Comment:
Can we use Jackson? I remember we were trying to remove other JSON dependencies such as Gson from Debezium BOM. cc @gunnarmorling

Reply:
What's the problem of JSON or GSON library?  I can convert the code to use Jackson's ObjectMapper, but I think the code will be more verbose.

Reply:
+1 try to avoid more than 1 json serialization library. This will also make graduating from incubation and into core easier.

Reply:
OK, let me convert the code using Jackson's ObjectMapper